model/user.go

```diff
@@ -82,6 +82,7 @@ type User struct {
 	Timezone               StringMap `json:"timezone"`
 	MfaActive              bool      `json:"mfa_active,omitempty"`
 	MfaSecret              string    `json:"mfa_secret,omitempty"`
+	MfaRecovery            string    `json:"mfa_recovery,omitempty"`
```

Comment:
Why are we storing the MFA recovery codes as JSON encoded?

Also we can't accept modifications to the users table. Can we store them in props?

Reply:
Ok, removing the `MfaRecovery` field, in favor of props.
Can you point me to a location in the app/code, where they're used, please?

Reply:
Actually I realized that's a bad idea, since the props are sent to the client.

Reply:
Will need to figure out somewhere else to store them.

Reply:
Ok, I'm open to suggestions

Reply:
Going with the separate MFA recovery codes table, as confirmed by @jespino